client/modules/food/components/packing/PackingList.js

```diff
@@ -107,7 +107,10 @@ class PackingList extends Component {
 
   handleSelectAll = isSelected => {
     const {selected} = this.state
-    const {customers} = this.props
+    const customers = []
+    this.props.packedCustomers.forEach((c) => {
```

Comment:
This can be done a bit nicer with `filter`, but it's not really important. If you keep it like this though I'd declare customers with `let customers = []` to show you intend to change it. (The actual array the customers variable points to doesn't change if you push to it, so const works fine. I think it's a bit more descriptive though, and it's that way in the rest of the code. Same for `unselectable` below.)